assets/scripts/src/choices.js

```diff
@@ -2348,6 +2347,14 @@ class Choices {
       dropdown.insertBefore(input, dropdown.firstChild);
     }
 
+    if (
+      this.placeholder &&
+      this.passedElement.type !== 'select-one' &&
+      !this.config.searchPlaceholderValue
+    ) {
+      input.style.width = getWidthOfInput(input);
```

Comment:
I wonder if we could achieve the same thing with `requestAnimationFrame()` - as we probably only need to wait 1 frame